ci-operator/config/openshift-priv/cluster-network-operator/openshift-priv-cluster-network-operator-release-4.16.yaml

```diff
@@ -93,6 +93,12 @@ tests:
   steps:
     cluster_profile: aws
     workflow: openshift-e2e-aws-ovn-serial
+- as: e2e-aws-ovn-ipsec-serial
+  optional: true
+  steps:
+    cluster_profile: aws
+    workflow: openshift-e2e-aws-ovn-ipsec-serial
```

Comment:
@jluhrsen The `TEST_SUITE` env is already present in `openshift-e2e-aws-ovn-ipsec-serial` workflow.  so this may be redundant if we pass this again. can i just remove it ?

Reply:
Yes, I guess you can remove it from the others as well?

Reply:
yes @jcaamano, removed it.